README.md

```diff
@@ -506,17 +540,20 @@ yarn install
 These scripts can be run via `yarn` or `npm run`:
 
 <!-- AUTO-GENERATED-CONTENT:START (SCRIPTS) -->
-| Script | Description |
-|--------|-------------|
-| `docs` | updates any auto-generated-content blocks in [Markdown](https://guides.github.com/features/mastering-markdown/) files |
-| `format` | format the application code |
-| `format:package` | `./lib/cli/index.js -w package.json` |
-| `format:source` | `prettier --write "**/*.{md,js,json}"` |
-| `gamut` | run the full gamut of checks - reset environment, generate docs, format and lint code, and run tests |
-| `lint` | lint the application code |
-| `prepublishOnly` | make sure the package is in good state before publishing |
-| `reset` | reset the `node_modules` dependencies |
-| `test` | run unit tests for the application |
+
+| Script           | Description                                                                                                           |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------- |
```

Comment:
Can we update the descriptions of the scripts below that are missing them? I screwed up and forgot to add them back in for the other `format` scripts.

Reply:
Done.